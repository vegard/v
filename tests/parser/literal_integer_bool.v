1100100100001111110110101010001000100001011010001100001000110100110001001100011001100010100010111000000011011100000111001101b
