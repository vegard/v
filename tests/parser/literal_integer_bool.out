(literal_integer 16703571626015105435307505830654230989)
